tools/start_notebook.sh

```diff
@@ -21,8 +21,19 @@ bazel build -c opt //temporian
 PKDIR="$(pwd)/build_package"
 rm -fr ${PKDIR}
 mkdir -p ${PKDIR}
-find temporian -name "*.py" -type f -exec cp --parents {} ${PKDIR}/ \;
-( cd bazel-bin && find temporian \( -name "*.so" -o -name "*.py" \) -type f -exec cp --parents {} ${PKDIR}/ \; )
+find temporian -name "*.py" -type f -exec sh -c '
+    dir="$(dirname "$1")"
+    mkdir -p "${2}/${dir}"
+    cp "$1" "${2}/${dir}/$(basename "$1")"
+  ' _ {} ${PKDIR} \;
+
+( cd bazel-bin && \
+  find temporian \( -name "*.so" -o -name "*.py" \) -type f -exec sh -c '
+    dir="$(dirname "$1")"
```

Comment:
This is complex :) I think rsync is available on Macos, can this be used instead?
Something like (I did not test):

( cd bazel-bin && \
find temporian \( -name "*.so" -o -name "*.py" \) -type f -exec rsync -R -arv {} ${PKDIR}/ \; )